app/views/declaration/draft_declaration_page.scala.html

```diff
@@ -30,15 +31,25 @@
     paragraph: paragraph
 )
 
-@()(implicit request: Request[_], flash: Flash, messages: Messages)
+@(declarationId: Option[String], expiry: String)(implicit request: Request[_], messages: Messages)
 
-@expiresMessage(date: String) = {
+@expiresMessage = {
     <div id="draft_confirmation-expiry" class="govuk-panel__body">
         @messages("declaration.draft.info")<br>
-        @ViewDates.formatDate(Instant.ofEpochMilli(date.toLong).atOffset(ZoneOffset.UTC))
+        @ViewDates.formatDate(Instant.ofEpochMilli(expiry.toLong).atOffset(ZoneOffset.UTC))
     </div>
 }
 
+@viewDeclarationSummaryLink = { @if(declarationId.isDefined) {
+  <p class="govuk-body">
+    @link(
+      id = Some("view_declaration_summary"),
+      text = messages("declaration.draft.viewSummary"),
+      call = SavedDeclarationsController.continueDeclaration(declarationId.get)
+    )
+  </p>
+}}
```

Comment:
NIT: you could just use a `.map` on the option instead of `isDefined` and then a `.get`